### PR TITLE
Add list dependencies command

### PR DESCRIPTION
**Pull request by belemaire**
Depends on https://github.com/electrode-io/electrode-native/pull/439
Please merge https://github.com/electrode-io/electrode-native/pull/439 before merging this one

### CODE REVIEW

**Comment by belemaire** [comment]
Build failing due to #439 not merged. Please ignore and merge once #439 is merged.

File: ern-local-cli/src/commands/list.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+// @flow
+
+exports.command = 'list'
+exports.desc = 'List information associated to an Electrode Native module'
```
Comment (krunalsshah) [inline ern-local-cli/src/commands/list.js]: Lists the native dependencies of an Electrode Native module

File: ern-local-cli/src/commands/list/dependencies.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  } catch (e) {
+    throw e
+  }
+}
+
+function logDependencies (dependencies: Array<Dependency>, type: string) {
+  if (!_.isEmpty(dependencies)) {
+    console.log(chalk.blue.bold(`=== ${type} ===`))
+    for (const d of dependencies) {
+      console.log(d.toString())
```
Comment (krunalsshah) [inline ern-local-cli/src/commands/list/dependencies.js]: log.d?

File: ern-local-cli/src/commands/list/dependencies.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    logDependencies(dependencies.thirdPartyInManifest, 'Third party declared in Manifest')
+    logDependencies(dependencies.thirdPartyNotInManifest, 'Third party not declared in Manifest')
+  } catch (e) {
+    throw e
+  }
+}
+
+function logDependencies (dependencies: Array<Dependency>, type: string) {
+  if (!_.isEmpty(dependencies)) {
+    console.log(chalk.blue.bold(`=== ${type} ===`))
```
Comment (krunalsshah) [inline ern-local-cli/src/commands/list/dependencies.js]: log.d?

**Comment by krunalsshah** [comment]
@belemaire looks this PR depends on the other refactor PR to merge.

**Comment by belemaire** [comment]
@krunalsshah Keeping `console.log(d.toString())` otherwise if I use `console.log(d)` it will log the string representation of the object (all the fields) as such :

```
Dependency {
  name: 'appium-android-bootstrap',
  scope: undefined,
  version: '2.10.0' }
```

which is not the desired output. Only way is to force evaluation to a string (so that it calls `toString()` properly), in the following way for example `console.log(''+d)`, which is less clear than explicit call to `toString()`.

**Pull request merged**